arwen/elrondapi/generate/eiGenWriteHeaders.go

```diff
@@ -0,0 +1,21 @@
+package elrondapigenerate
+
+import (
+	"fmt"
+)
+
+func autoGeneratedHeader(out *eiGenWriter) {
```

Comment:
There are some left over places where we can use this.

Reply:
Yes, but I'd rather fix this in the next PR